src/base/stats/owl_base_stats.mli

```diff
@@ -133,6 +133,8 @@ val pp_hist: Format.formatter -> histogram -> unit
 val tukey_fences : ?k:float -> float array -> float * float
 (** Refer to :doc:`owl_stats`. *)
 
+val iqr : float array -> float
```

Comment:
Because we have used `first_quartile` and `third_quartile` as functions names, it makes more sense to use full name (i.e. `interquartile`) imo.